docs/notes/bufgix-17624.md

```diff
@@ -0,0 +1 @@
+#The files and folders functions crash on Android
```

Comment:
@montegoulding Two issues here:

- The release note is named incorrectly (the filename is `bufgix` when it needs to be `bugfix`)
- Please write the release note in a positive way (e.g. "Fix crash on Android in the files and folders functions")